src/main/kotlin/org/rust/ide/refactoring/inlineFunction/RsInlineFunctionProcessor.kt

```diff
@@ -263,6 +314,26 @@ class RsInlineFunctionProcessor(
         }
     }
 
+    private fun replaceParamWithExpr(param: RsPat, argument: RsExpr, funcScope: LocalSearchScope) {
```

Comment:
It seems that this functions is unused?

Reply:
It isn't used, since we decided to always make a *let* declaration, and will be used again if we won't create it when unecessary besides the obvious `name != argument.text` ([explained here](https://github.com/intellij-rust/intellij-rust/pull/6004#discussion_r512216550)) that's already covered. I can remove it for now too if you want.

Reply:
Let's remove it for this PR then :)

Reply:
Done